script/seed.js

```diff
@@ -1,18 +1,34 @@
 'use strict'
 
 const db = require('../server/db')
-const {User} = require('../server/db/models')
+const { User, Orders, Reviews, Sunglasses } = require('../server/db/models')
+
 
 async function seed() {
   await db.sync({force: true})
   console.log('db synced!')
 
   const users = await Promise.all([
-    User.create({email: 'cody@email.com', password: '123'}),
-    User.create({email: 'murphy@email.com', password: '123'})
+    User.create({name:"cody",email: 'cody@email.com', password: '123'}),
+    User.create({name:"murphy", email: 'murphy@email.com', password: '123'})
   ])
 
+  const orders = await Promise.all([
+    Orders.create({quantity:1, price:500, timestamp:Date.now(), status:'Created'})
```

Comment:
add foreign key for user that order belongs to ('userId'); manually add (method is along the lines of 'addSunglasses') to orders